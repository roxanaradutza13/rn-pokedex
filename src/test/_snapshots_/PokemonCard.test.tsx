import { render } from "@testing-library/react-native";
import { View, Text } from "react-native";
import renderer from 'react-test-renderer';

test('component View mounts properly', () => {
    const wrapper = render(<View />)
    expect(wrapper).toBeTruthy()
})
test('component Text with Pokemon wartortle mounts properly', () => {
    const wrapper = render(<Text>wartortle</Text>)
    expect(wrapper).toBeTruthy()
})
test('component Text with Pokemon wartortle id #8 mounts properly', () => {
    const wrapper = render(<Text>#8</Text>)
    expect(wrapper).toBeTruthy()
})

test('SnapShot Text wartortle correctly', () => {
    const tree = renderer.create(<Text>wartortle</Text>).toJSON();
    expect(tree).toMatchSnapshot();
});