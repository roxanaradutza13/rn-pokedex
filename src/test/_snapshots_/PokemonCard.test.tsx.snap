// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapShot Text wartortle correctly 1`] = `
<Text>
  wartortle
</Text>
`;